### ISSUE TITLE
PDF Export / Import not correct

### ISSUE THREAD

**Issue by lisandi** [ether/ep_file_menu_toolbar#20]
The export to PDF functionality does not export the formatting of a text and also ads the user in a color actually not even assigned to the user - if it should get added at all - Instead it would be tetter to add only the contributors on top or underlining.
[New.pdf](https://github.com/ether/ep_file_menu_toolbar/files/7266568/New.pdf)
That is the page that got exported! The same problem with ODT 
![Screen Shot 2021-10-01 at 17 46 29](https://user-images.githubusercontent.com/1982011/135607695-b2b09e27-31f5-49d8-b7c4-38b47ffdef5b.png)
Other export formats look as follow - on the same page:
Github doesn't like all formats so .txt got added and needs to get removed first.

<hr>
DOC - [New.doc.txt](https://github.com/ether/ep_file_menu_toolbar/files/7266652/New.doc.txt)
ETHERPAD - [New.etherpad.txt](https://github.com/ether/ep_file_menu_toolbar/files/7266655/New.etherpad.txt)
HTML - [New.html.txt](https://github.com/ether/ep_file_menu_toolbar/files/7266660/New.html.txt)
MD - [New.md](https://github.com/ether/ep_file_menu_toolbar/files/7266669/New.md)
ODT - [New.odt](https://github.com/ether/ep_file_menu_toolbar/files/7266670/New.odt)
PDF - [New.pdf](https://github.com/ether/ep_file_menu_toolbar/files/7266568/New.pdf)
TXT - [New.txt](https://github.com/ether/ep_file_menu_toolbar/files/7266674/New.txt)

Importing the doc, odt, md document which had been exported results first in a red error message while still loading the document 
![Screen Shot 2021-10-01 at 17 50 30](https://user-images.githubusercontent.com/1982011/135608648-4ac8b243-d379-4af0-807a-f3aebee653bc.png)
The loaded Document has no formatting
![Screen Shot 2021-10-01 at 17 51 12](https://user-images.githubusercontent.com/1982011/135608687-1f78746e-d252-4cb4-a7fa-6621345c7da5.png)
The import of the exported etherpad format works fine, when no author had been exported it will set the author to null
![Screen Shot 2021-10-01 at 17 52 10](https://user-images.githubusercontent.com/1982011/135608778-41714e0a-4cab-4a63-82bc-37ae2acedd3c.png)
Making adding an author name when entering the pad would be advised to avoid that problem.

When importing the Etherpad format the file menu bar gets no more displayed at the end of the import and it will come back after reloading the page. - Perhaps it would be helpful to reload the page after an import by default to avoid a missing file-menu-bar.
![Screen Shot 2021-10-01 at 18 09 38](https://user-images.githubusercontent.com/1982011/135610744-da010b55-0b40-4d96-b5c3-3084a7b2d044.png)
The file menu is actually still there after an import but has no content anymore - you can check when you move the mouse to the top until the dropdowns of the menu show up
(![Screen Shot 2021-10-01 at 18 12 27](https://user-images.githubusercontent.com/1982011/135610900-1d09f794-cb75-478a-816d-196f2fde43f3.png)
empty) 

Importing the odt format results in an open office style looking button bar :-) but the text format gets lost.
![Screen Shot 2021-10-01 at 18 23 53](https://user-images.githubusercontent.com/1982011/135612232-3c4fc450-9580-4b55-8167-ba25ad811cd3.png)
... and the odt button bar gets lost after a page reload 
- also here it would be advised to have an automated reload after a page import

The MD import looks fine but during the export the line spacing and the alignment got lost and paragraphs got added a * at the beginning and all that won't get replaced when imported, while headings (beside H5 and H6 which no exist until now) and all list formats get imported OK (dl) not sure!

I would therefore add MD format to the default export formats in the filemenu menu - it would avoid having that plugin to be installed first. MD is actually the only format which comes pretty near to the etherpad format - all other failed!

![Screen Shot 2021-10-01 at 18 35 16](https://user-images.githubusercontent.com/1982011/135613524-d9265aa1-ae15-44b4-9874-0159870543c3.png)

**Comment by JohnMcLear** [comment]
Each plugin is responsible for the formatting type within each format IE PDF.

You would need to check each plugin IE headings/font size/color to ensure it's working properly and then post on those repos.

**Comment by lisandi** [comment]
So I need to post all in the core right as ONLY the MD is another plugin and that exports/imports the best actually. I will copy that over and post that MD on the MD Plugin the rest in core. - The point is it is not at all clear where the red error (a core message actually) comes from and if it is a plugin thing or a core thing or a problem of the file menu as with its default links I tested everything. 

By the way, I am also doing that only in my free time but right now a bit more as we see potential in integrating that for the rural schools we have in mind in Africa and Asia. It takes also time to write down and discover everything - problems that actually should not exist at first place. The main problem is mainly actually the fact that many plugins listed in the admin settings are simply not at all working - some have even not even one line of code!!! - Having here a bit more systematic would definitely help to get that clutter and mess sorted out and ONLY those plugins which work also in a complete install with all other plugins would get listed there. That again would challenge but also encourage developers of plugins to test their plugins not only with a blank install but also with a very very full feature install to ensure that their plugins can work with each other. If that install would even be a subfolder one also the relative path problem would be solved easily.

I am happy to set up a site - actually doing it anyway here for us - to test all the plugins with each other. If they can't work with each other it must be somehow marked (by the developer!!! and not by the users who often have no programming knowledge but think they can install a plugin from the admin panel!) 

The colors, headings, font size, and etc plugins are working just fine as you can see in the etherpad import/export! 

I hope you see the problem in organization - as you can't assume from people installing a plugin from admin/plugin that they can code at all! Nor they would even know how to solve a crashed etherpad! Etherpad is a great tool but the way the plugins get organized is a complete mess! Sorry to say that!

At least those work together here - more to come:

```
Etherpad version
Version number: 1.8.14

Latest available version: 1.8.14

Git sha: d262e31

Installed plugins
ep_aa_file_menu_toolbar
ep_adminpads2
ep_align
ep_announce
ep_author_neat2
ep_author_hover
ep_button_link
ep_author_follow
ep_clear_formatting
ep_comments_page
ep_code_formatting
ep_copy_paste_select_all
ep_custom_inline_toolbar
ep_define
ep_desktop_notifications
ep_embedded_hyperlinks2
ep_embedmedia
ep_etherpad-lite
ep_headings2
ep_font_color
ep_font_family
ep_font_size
ep_image_upload
ep_help_bubbles
ep_markdown
ep_line_spacing
ep_mathjax
ep_padlist2
ep_set_title_on_pad
ep_spellcheck
ep_stats
ep_subscript_and_superscript
ep_table_of_contents
ep_template_content
ep_timestamp
ep_webrtc
ep_workspaces
ep_readonly_guest
Installed parts
ep_readonly_guest/ep_readonly_guest
ep_workspaces/Workspaces
ep_webrtc/ep_webrtc
ep_timestamp/ep_timestamp
ep_template_content/main
ep_table_of_contents/ep_table_of_contents
ep_subscript_and_superscript/main
ep_stats/stats
ep_spellcheck/ep_spellcheck
ep_set_title_on_pad/set_title_on_pad
ep_padlist2/ep_padlist2
ep_mathjax/mathjax
ep_line_spacing/main
ep_markdown/markdown
ep_help_bubbles/help_bubbles
ep_image_upload/ep_image_upload
ep_font_size/main
ep_font_family/main
ep_font_color/main
ep_headings2/main
ep_etherpad-lite/openapi
ep_etherpad-lite/adminsettings
ep_etherpad-lite/adminplugins
ep_etherpad-lite/admin
ep_etherpad-lite/tests
ep_etherpad-lite/socketio
ep_etherpad-lite/errorhandling
ep_etherpad-lite/importexport
ep_etherpad-lite/apicalls
ep_etherpad-lite/padurlsanitize
ep_etherpad-lite/specialpages
ep_etherpad-lite/i18n
ep_etherpad-lite/stats
ep_etherpad-lite/express
ep_etherpad-lite/Minify
ep_etherpad-lite/DB
ep_embedmedia/embedmedia
ep_embedded_hyperlinks2/main
ep_desktop_notifications/ep_desktop_notifications
ep_define/ep_define
ep_custom_inline_toolbar/custom_inline_toolbar
ep_copy_paste_select_all/copy_paste_select_all
ep_code_formatting/ep_code_formatting
ep_etherpad-lite/webaccess
ep_comments_page/comments_page
ep_etherpad-lite/static
ep_clear_formatting/clear_formatting
ep_author_follow/ep_author_follow
ep_button_link/button_link
ep_author_hover/ep_author_hover
ep_author_neat2/ep_author_neat2
ep_announce/ep_announce
ep_align/main
ep_adminpads2/ep_adminpads2
ep_aa_file_menu_toolbar/aa_file_menu_toolbar
Installed hooks
Server-side hooks
aceAttribClasses:
ep_font_family:
main:
ep_font_family/index
ep_subscript_and_superscript:
main:
ep_subscript_and_superscript/index
asyncLineHTMLForExport:
ep_subscript_and_superscript:
main:
ep_subscript_and_superscript/index
authenticate:
ep_readonly_guest:
ep_readonly_guest:
ep_readonly_guest
ccRegisterBlockElements:
ep_image_upload:
ep_image_upload:
ep_image_upload/static/js/contentCollection
clientVars:
ep_author_follow:
ep_author_follow:
ep_author_follow/index:clientVars
ep_button_link:
button_link:
ep_comments_page:
comments_page:
ep_comments_page/index
ep_custom_inline_toolbar:
custom_inline_toolbar:
ep_custom_inline_toolbar/index
ep_image_upload:
ep_image_upload:
ep_image_upload/index
ep_readonly_guest:
ep_readonly_guest:
ep_readonly_guest
ep_set_title_on_pad:
set_title_on_pad:
ep_set_title_on_pad/handleMessage
ep_webrtc:
ep_webrtc:
collectContentImage:
ep_image_upload:
ep_image_upload:
ep_image_upload/static/js/contentCollection
collectContentPost:
ep_align:
main:
ep_align/static/js/shared
ep_font_color:
main:
ep_font_color/static/js/shared
ep_font_family:
main:
ep_font_family/static/js/shared
ep_headings2:
main:
ep_headings2/static/js/shared
ep_image_upload:
ep_image_upload:
ep_image_upload/static/js/contentCollection
ep_mathjax:
mathjax:
ep_mathjax/static/js/shared
ep_subscript_and_superscript:
main:
ep_subscript_and_superscript/static/js/shared
collectContentPre:
ep_align:
main:
ep_align/static/js/shared
ep_comments_page:
comments_page:
ep_comments_page/static/js/shared
ep_font_color:
main:
ep_font_color/static/js/shared
ep_font_family:
main:
ep_font_family/static/js/shared
ep_font_size:
main:
ep_font_size/static/js/shared
ep_headings2:
main:
ep_headings2/static/js/shared
ep_image_upload:
ep_image_upload:
ep_image_upload/static/js/contentCollection
ep_mathjax:
mathjax:
ep_mathjax/static/js/shared
ep_subscript_and_superscript:
main:
ep_subscript_and_superscript/static/js/shared
createServer:
ep_etherpad-lite:
express:
ep_etherpad-lite/node/hooks/express
eejsBlock_adminMenu:
ep_adminpads2:
ep_adminpads2:
ep_adminpads2
eejsBlock_body:
ep_aa_file_menu_toolbar:
aa_file_menu_toolbar:
ep_aa_file_menu_toolbar/eejs
ep_custom_inline_toolbar:
custom_inline_toolbar:
ep_custom_inline_toolbar/index
ep_embedmedia:
embedmedia:
ep_embedmedia/hooks:eejsBlock_body
ep_image_upload:
ep_image_upload:
ep_image_upload/index
ep_set_title_on_pad:
set_title_on_pad:
ep_set_title_on_pad/eejs
ep_stats:
stats:
ep_stats/index
ep_template_content:
main:
ep_template_content/client
eejsBlock_dd_edit:
ep_copy_paste_select_all:
copy_paste_select_all:
ep_copy_paste_select_all/copy_paste_select_all
eejsBlock_dd_format:
ep_clear_formatting:
clear_formatting:
ep_clear_formatting/clear_formatting
ep_font_color:
main:
ep_font_color/index
ep_font_family:
main:
ep_font_family/index
ep_font_size:
main:
ep_font_size/index
ep_subscript_and_superscript:
main:
ep_subscript_and_superscript/index
eejsBlock_dd_help:
ep_define:
ep_define:
ep_define/eejs
eejsBlock_dd_view:
ep_spellcheck:
ep_spellcheck:
ep_spellcheck/spellcheck
ep_stats:
stats:
ep_stats/index
ep_table_of_contents:
ep_table_of_contents:
ep_table_of_contents/index
eejsBlock_editbarMenuLeft:
ep_align:
main:
ep_align/index
ep_comments_page:
comments_page:
ep_comments_page/index
ep_embedded_hyperlinks2:
main:
ep_embedded_hyperlinks2/index
ep_embedmedia:
embedmedia:
ep_embedmedia/hooks:eejsBlock_editbarMenuLeft
ep_font_color:
main:
ep_font_color/index
ep_font_family:
main:
ep_font_family/index
ep_font_size:
main:
ep_font_size/index
ep_headings2:
main:
ep_headings2/index
ep_image_upload:
ep_image_upload:
ep_image_upload/editbar
ep_line_spacing:
main:
ep_line_spacing/index
ep_mathjax:
mathjax:
ep_mathjax/hooks:eejsBlock_editbarMenuLeft
ep_subscript_and_superscript:
main:
ep_subscript_and_superscript/index
ep_template_content:
main:
ep_template_content/client
ep_timestamp:
ep_timestamp:
ep_timestamp/timestamp
eejsBlock_editbarMenuRight:
ep_help_bubbles:
help_bubbles:
ep_help_bubbles/static/js/help_bubbles.js:eejsBlock_editbarMenuRight
eejsBlock_editorContainerBox:
ep_embedded_hyperlinks2:
main:
ep_embedded_hyperlinks2/index
ep_mathjax:
mathjax:
ep_mathjax/hooks:eejsBlock_editorContainerBox
ep_table_of_contents:
ep_table_of_contents:
ep_table_of_contents/index
eejsBlock_exportColumn:
ep_markdown:
markdown:
ep_markdown/index
ep_stats:
stats:
ep_stats/index
eejsBlock_indexWrapper:
ep_padlist2:
ep_padlist2:
ep_padlist2
eejsBlock_mySettings:
ep_author_follow:
ep_author_follow:
ep_author_follow/index
ep_author_hover:
ep_author_hover:
ep_author_hover/index
ep_comments_page:
comments_page:
ep_comments_page/index
ep_custom_inline_toolbar:
custom_inline_toolbar:
ep_custom_inline_toolbar/index
ep_define:
ep_define:
ep_define/eejs
ep_desktop_notifications:
ep_desktop_notifications:
ep_desktop_notifications/desktop_notifications
ep_markdown:
markdown:
ep_markdown/index
ep_set_title_on_pad:
set_title_on_pad:
ep_set_title_on_pad/index
ep_spellcheck:
ep_spellcheck:
ep_spellcheck/spellcheck
ep_stats:
stats:
ep_stats/index
ep_table_of_contents:
ep_table_of_contents:
ep_table_of_contents/index
ep_webrtc:
ep_webrtc:
eejsBlock_permissionDenied:
ep_readonly_guest:
ep_readonly_guest:
ep_readonly_guest
eejsBlock_scripts:
ep_comments_page:
comments_page:
ep_comments_page/index
ep_embedmedia:
embedmedia:
ep_embedmedia/hooks:eejsBlock_scripts
ep_help_bubbles:
help_bubbles:
ep_help_bubbles/static/js/help_bubbles.js:eejsBlock_scripts
ep_markdown:
markdown:
ep_markdown/index
ep_mathjax:
mathjax:
ep_mathjax/hooks:eejsBlock_scripts
ep_stats:
stats:
ep_stats/index
ep_table_of_contents:
ep_table_of_contents:
ep_table_of_contents/index
ep_timestamp:
ep_timestamp:
ep_timestamp/timestamp
eejsBlock_styles:
ep_aa_file_menu_toolbar:
aa_file_menu_toolbar:
ep_aa_file_menu_toolbar/eejs
ep_comments_page:
comments_page:
ep_comments_page/index
ep_custom_inline_toolbar:
custom_inline_toolbar:
ep_custom_inline_toolbar/index
ep_embedmedia:
embedmedia:
ep_embedmedia/hooks:eejsBlock_styles
ep_help_bubbles:
help_bubbles:
ep_help_bubbles/static/js/help_bubbles.js:eejsBlock_styles
ep_image_upload:
ep_image_upload:
ep_image_upload/index
ep_mathjax:
mathjax:
ep_mathjax/hooks:eejsBlock_styles
ep_stats:
stats:
ep_stats/index
ep_table_of_contents:
ep_table_of_contents:
ep_table_of_contents/index
ep_template_content:
main:
ep_template_content/client
ep_timestamp:
ep_timestamp:
ep_timestamp/timestamp
ep_webrtc:
ep_webrtc:
eejsBlock_userlist:
ep_readonly_guest:
ep_readonly_guest:
ep_readonly_guest
exportEtherpadAdditionalContent:
ep_comments_page:
comments_page:
ep_comments_page/index
exportFileName:
ep_set_title_on_pad:
set_title_on_pad:
ep_set_title_on_pad/index
exportHTMLAdditionalContent:
ep_comments_page:
comments_page:
ep_comments_page/exportHTML
exportHtmlAdditionalTags:
ep_font_family:
main:
ep_font_family/index
ep_subscript_and_superscript:
main:
ep_subscript_and_superscript/index
exportHtmlAdditionalTagsWithData:
ep_comments_page:
comments_page:
ep_comments_page/exportHTML
ep_font_color:
main:
ep_font_color/exportHTML
ep_font_size:
main:
ep_font_size/exportHTML
expressCloseServer:
ep_etherpad-lite:
socketio:
ep_etherpad-lite/node/hooks/express/socketio
expressConfigure:
ep_etherpad-lite:
webaccess:
ep_etherpad-lite/node/hooks/express/webaccess
ep_image_upload:
ep_image_upload:
ep_image_upload/index
expressCreateServer:
ep_adminpads2:
ep_adminpads2:
ep_adminpads2
ep_comments_page:
comments_page:
ep_comments_page/index
ep_etherpad-lite:
admin:
ep_etherpad-lite/node/hooks/express/admin
adminplugins:
ep_etherpad-lite/node/hooks/express/adminplugins
adminsettings:
ep_etherpad-lite/node/hooks/express/adminsettings
apicalls:
ep_etherpad-lite/node/hooks/express/apicalls
errorhandling:
ep_etherpad-lite/node/hooks/express/errorhandling
i18n:
ep_etherpad-lite/node/hooks/i18n
importexport:
ep_etherpad-lite/node/hooks/express/importexport
openapi:
ep_etherpad-lite/node/hooks/express/openapi
padurlsanitize:
ep_etherpad-lite/node/hooks/express/padurlsanitize
socketio:
ep_etherpad-lite/node/hooks/express/socketio
specialpages:
ep_etherpad-lite/node/hooks/express/specialpages
static:
ep_etherpad-lite/node/hooks/express/static
tests:
ep_etherpad-lite/node/hooks/express/tests
ep_markdown:
markdown:
ep_markdown/express
ep_padlist2:
ep_padlist2:
ep_padlist2
ep_readonly_guest:
ep_readonly_guest:
ep_readonly_guest
ep_stats:
stats:
ep_stats/express
ep_workspaces:
Workspaces:
ep_workspaces/js/express.js
getLineHTMLForExport:
ep_align:
main:
ep_align/index
ep_comments_page:
comments_page:
ep_comments_page/exportHTML
ep_font_color:
main:
ep_font_color/exportHTML
ep_font_family:
main:
ep_font_family/index
ep_font_size:
main:
ep_font_size/exportHTML
ep_headings2:
main:
ep_headings2/index
ep_image_upload:
ep_image_upload:
ep_image_upload/exportHTML
handleMessage:
ep_define:
ep_define:
ep_define/handleMessage
ep_readonly_guest:
ep_readonly_guest:
ep_readonly_guest
ep_set_title_on_pad:
set_title_on_pad:
ep_set_title_on_pad/handleMessage
ep_webrtc:
ep_webrtc:
handleMessageSecurity:
ep_comments_page:
comments_page:
ep_comments_page/index
import:
ep_markdown:
markdown:
ep_markdown/index
loadSettings:
ep_button_link:
button_link:
ep_custom_inline_toolbar:
custom_inline_toolbar:
ep_custom_inline_toolbar/index
ep_image_upload:
ep_image_upload:
ep_image_upload/settings
ep_readonly_guest:
ep_readonly_guest:
ep_readonly_guest
ep_webrtc:
ep_webrtc:
padCopy:
ep_comments_page:
comments_page:
ep_comments_page/index
padCreate:
ep_adminpads2:
ep_adminpads2:
ep_adminpads2
padInitToolbar:
ep_align:
main:
ep_align/index
ep_code_formatting:
ep_code_formatting:
ep_code_formatting/index
ep_comments_page:
comments_page:
ep_comments_page/index
ep_custom_inline_toolbar:
custom_inline_toolbar:
ep_custom_inline_toolbar/index
ep_font_color:
main:
ep_font_color/index
ep_font_size:
main:
ep_font_size/index
ep_line_spacing:
main:
ep_line_spacing/index
ep_subscript_and_superscript:
main:
ep_subscript_and_superscript/index
padRemove:
ep_adminpads2:
ep_adminpads2:
ep_adminpads2
ep_comments_page:
comments_page:
ep_comments_page/index
ep_image_upload:
ep_image_upload:
ep_image_upload/index
preAuthorize:
ep_readonly_guest:
ep_readonly_guest:
ep_readonly_guest
restartServer:
ep_etherpad-lite:
express:
ep_etherpad-lite/node/hooks/express
shutdown:
ep_etherpad-lite:
DB:
ep_etherpad-lite/node/db/DB
express:
ep_etherpad-lite/node/hooks/express
Minify:
ep_etherpad-lite/node/utils/Minify
stats:
ep_etherpad-lite/node/stats
socketio:
ep_adminpads2:
ep_adminpads2:
ep_adminpads2
ep_comments_page:
comments_page:
ep_comments_page/index
ep_etherpad-lite:
adminplugins:
ep_etherpad-lite/node/hooks/express/adminplugins
adminsettings:
ep_etherpad-lite/node/hooks/express/adminsettings
socketio:
ep_etherpad-lite/node/handler/PadMessageHandler
ep_webrtc:
ep_webrtc:
:setSocketIO
ep_workspaces:
Workspaces:
ep_workspaces/js/socketio.js
stylesForExport:
ep_font_color:
main:
ep_font_color/exportHTML
ep_font_size:
main:
ep_font_size/exportHTML
ep_headings2:
main:
ep_headings2/index
ep_image_upload:
ep_image_upload:
ep_image_upload/exportHTML
Client-side hooks
aceAttribClasses:
ep_code_formatting:
ep_code_formatting:
ep_code_formatting/static/js/index
ep_font_family:
main:
ep_font_family/static/js/index
ep_subscript_and_superscript:
main:
ep_subscript_and_superscript/static/js/index
aceAttribsToClasses:
ep_align:
main:
ep_align/static/js/index
ep_code_formatting:
ep_code_formatting:
ep_code_formatting/static/js/index
ep_comments_page:
comments_page:
ep_comments_page/static/js/index
ep_embedded_hyperlinks2:
main:
ep_embedded_hyperlinks2/static/js/index
ep_embedmedia:
embedmedia:
ep_embedmedia/static/js/hooks:aceAttribsToClasses
ep_font_color:
main:
ep_font_color/static/js/index
ep_font_family:
main:
ep_font_family/static/js/index
ep_font_size:
main:
ep_font_size/static/js/index
ep_headings2:
main:
ep_headings2/static/js/index
ep_image_upload:
ep_image_upload:
ep_image_upload/static/js/clientHooks
ep_line_spacing:
main:
ep_line_spacing/static/js/index
ep_mathjax:
mathjax:
ep_mathjax/static/js/hooks:aceAttribsToClasses
ep_subscript_and_superscript:
main:
ep_subscript_and_superscript/static/js/index
aceCreateDomLine:
ep_code_formatting:
ep_code_formatting:
ep_code_formatting/static/js/index
ep_embedded_hyperlinks2:
main:
ep_embedded_hyperlinks2/static/js/index
ep_embedmedia:
embedmedia:
ep_embedmedia/static/js/hooks:aceCreateDomLine
ep_font_color:
main:
ep_font_color/static/js/index
ep_font_size:
main:
ep_font_size/static/js/index
ep_mathjax:
mathjax:
ep_mathjax/static/js/hooks:aceCreateDomLine
aceDomLineProcessLineAttributes:
ep_align:
main:
ep_align/static/js/index
ep_headings2:
main:
ep_headings2/static/js/index
ep_image_upload:
ep_image_upload:
ep_image_upload/static/js/clientHooks
ep_line_spacing:
main:
ep_line_spacing/static/js/index
aceEditEvent:
ep_align:
main:
ep_align/static/js/index
ep_author_neat2:
ep_author_neat2:
ep_author_neat2/static/js/index
ep_comments_page:
comments_page:
ep_comments_page/static/js/index
ep_font_color:
main:
ep_font_color/static/js/index
ep_font_family:
main:
ep_font_family/static/js/index
ep_headings2:
main:
ep_headings2/static/js/index
ep_stats:
stats:
ep_stats/static/js/stats
ep_subscript_and_superscript:
main:
ep_subscript_and_superscript/static/js/index
ep_table_of_contents:
ep_table_of_contents:
ep_table_of_contents/static/js/aceEditEvent:aceEditEvent
aceEditorCSS:
ep_code_formatting:
ep_code_formatting:
ep_code_formatting/static/js/index
ep_comments_page:
comments_page:
ep_comments_page/static/js/index
ep_custom_inline_toolbar:
custom_inline_toolbar:
ep_custom_inline_toolbar/static/js/index
ep_font_color:
main:
ep_font_color/static/js/index
ep_font_family:
main:
ep_font_family/static/js/index
ep_font_size:
main:
ep_font_size/static/js/index
ep_headings2:
main:
ep_headings2/static/js/index
ep_image_upload:
ep_image_upload:
ep_image_upload/static/js/clientHooks
ep_markdown:
markdown:
ep_markdown/static/js/markdown
aceInitialized:
ep_align:
main:
ep_align/static/js/index
ep_button_link:
button_link:
ep_button_link/static/js/main:postAceInit
ep_comments_page:
comments_page:
ep_comments_page/static/js/index
ep_custom_inline_toolbar:
custom_inline_toolbar:
ep_custom_inline_toolbar/static/js/index
ep_embedded_hyperlinks2:
main:
ep_embedded_hyperlinks2/static/js/index
ep_font_color:
main:
ep_font_color/static/js/index
ep_font_size:
main:
ep_font_size/static/js/index
ep_headings2:
main:
ep_headings2/static/js/index
ep_image_upload:
ep_image_upload:
ep_image_upload/static/js/clientHooks
ep_line_spacing:
main:
ep_line_spacing/static/js/index
ep_mathjax:
mathjax:
ep_mathjax/static/js/hooks:aceInitialized
aceInitInnerdocbodyHead:
ep_embedmedia:
embedmedia:
ep_embedmedia/static/js/hooks:aceInitInnerdocbodyHead
ep_mathjax:
mathjax:
ep_mathjax/static/js/hooks:aceInitInnerdocbodyHead
acePostWriteDomLineHTML:
ep_author_neat2:
ep_author_neat2:
ep_author_neat2/static/js/index
aceRegisterBlockElements:
ep_align:
main:
ep_align/static/js/index
ep_font_family:
main:
ep_font_family/static/js/index
ep_headings2:
main:
ep_headings2/static/js/index
ep_image_upload:
ep_image_upload:
ep_image_upload/static/js/clientHooks
ep_subscript_and_superscript:
main:
ep_subscript_and_superscript/static/js/index
aceSelectionChanged:
ep_code_formatting:
ep_code_formatting:
ep_code_formatting/static/js/index
ep_custom_inline_toolbar:
custom_inline_toolbar:
ep_custom_inline_toolbar/static/js/index
aceSetAuthorStyle:
ep_author_neat2:
ep_author_neat2:
ep_author_neat2/static/js/index
ccRegisterBlockElements:
ep_image_upload:
ep_image_upload:
ep_image_upload/static/js/contentCollection
chatNewMessage:
ep_desktop_notifications:
ep_desktop_notifications:
ep_desktop_notifications/static/js/desktop_notifications:chatNewMessage
collectContentImage:
ep_image_upload:
ep_image_upload:
ep_image_upload/static/js/contentCollection
collectContentPost:
ep_code_formatting:
ep_code_formatting:
ep_code_formatting/static/js/index
ep_font_family:
main:
ep_font_family/static/js/shared
ep_image_upload:
ep_image_upload:
ep_image_upload/static/js/contentCollection
ep_line_spacing:
main:
ep_line_spacing/static/js/shared
ep_mathjax:
mathjax:
ep_mathjax/static/js/shared
ep_subscript_and_superscript:
main:
ep_subscript_and_superscript/static/js/shared
collectContentPre:
ep_align:
main:
ep_align/static/js/shared
ep_code_formatting:
ep_code_formatting:
ep_code_formatting/static/js/index
ep_comments_page:
comments_page:
ep_comments_page/static/js/shared
ep_embedded_hyperlinks2:
main:
ep_embedded_hyperlinks2/static/js/index
ep_font_color:
main:
ep_font_color/static/js/shared
ep_font_family:
main:
ep_font_family/static/js/shared
ep_font_size:
main:
ep_font_size/static/js/shared
ep_headings2:
main:
ep_headings2/static/js/shared
ep_image_upload:
ep_image_upload:
ep_image_upload/static/js/contentCollection
ep_line_spacing:
main:
ep_line_spacing/static/js/shared
ep_mathjax:
mathjax:
ep_mathjax/static/js/shared
ep_subscript_and_superscript:
main:
ep_subscript_and_superscript/static/js/shared
handleClientMessage_ACCEPT_COMMIT:
ep_author_follow:
ep_author_follow:
ep_author_follow/static/js/follow:handleClientMessage_ACCEPT_COMMIT
ep_table_of_contents:
ep_table_of_contents:
ep_table_of_contents/static/js/aceEditEvent:aceEditEvent
handleClientMessage_CUSTOM:
ep_define:
ep_define:
ep_define/static/js/index
ep_set_title_on_pad:
set_title_on_pad:
ep_set_title_on_pad/static/js/index
handleClientMessage_NEW_CHANGES:
ep_author_follow:
ep_author_follow:
ep_author_follow/static/js/follow:handleClientMessage_NEW_CHANGES
ep_table_of_contents:
ep_table_of_contents:
ep_table_of_contents/static/js/aceEditEvent:aceEditEvent
handleClientMessage_RTC_MESSAGE:
ep_webrtc:
ep_webrtc:
ep_webrtc/static/js
postAceInit:
ep_aa_file_menu_toolbar:
aa_file_menu_toolbar:
ep_aa_file_menu_toolbar/static/js/index:documentReady
ep_align:
main:
ep_align/static/js/index
ep_announce:
ep_announce:
ep_announce/static/js/announce:announce.postAceInit
ep_author_follow:
ep_author_follow:
ep_author_follow/static/js/follow:postAceInit
ep_author_hover:
ep_author_hover:
ep_author_hover/static/js/index
ep_author_neat2:
ep_author_neat2:
ep_author_neat2/static/js/index
ep_clear_formatting:
clear_formatting:
ep_clear_formatting/static/js/clear_formatting
ep_comments_page:
comments_page:
ep_comments_page/static/js/index
ep_copy_paste_select_all:
copy_paste_select_all:
ep_copy_paste_select_all/static/js/copy_paste_select_all
ep_custom_inline_toolbar:
custom_inline_toolbar:
ep_custom_inline_toolbar/static/js/index
ep_define:
ep_define:
ep_define/static/js/index:postAceInit
ep_desktop_notifications:
ep_desktop_notifications:
ep_desktop_notifications/static/js/desktop_notifications:postAceInit
ep_embedded_hyperlinks2:
main:
ep_embedded_hyperlinks2/static/js/index
ep_font_color:
main:
ep_font_color/static/js/index
ep_font_family:
main:
ep_font_family/static/js/index
ep_font_size:
main:
ep_font_size/static/js/index
ep_headings2:
main:
ep_headings2/static/js/index
ep_line_spacing:
main:
ep_line_spacing/static/js/index
ep_markdown:
markdown:
ep_markdown/static/js/markdown
ep_mathjax:
mathjax:
ep_mathjax/static/js/hooks:postAceInit
ep_set_title_on_pad:
set_title_on_pad:
ep_set_title_on_pad/static/js/index:documentReady
ep_spellcheck:
ep_spellcheck:
ep_spellcheck/static/js/spellcheck:postAceInit
ep_stats:
stats:
ep_stats/static/js/stats
ep_subscript_and_superscript:
main:
ep_subscript_and_superscript/static/js/index
ep_table_of_contents:
ep_table_of_contents:
ep_table_of_contents/static/js/postAceInit:postAceInit
ep_template_content:
main:
ep_template_content/static/js/index
ep_timestamp:
ep_timestamp:
ep_timestamp/static/js/postAceInit:postAceInit
ep_webrtc:
ep_webrtc:
ep_webrtc/static/js
postToolbarInit:
ep_align:
main:
ep_align/static/js/index
ep_code_formatting:
ep_code_formatting:
ep_code_formatting/static/js/index
ep_comments_page:
comments_page:
ep_comments_page/static/js/index
ep_custom_inline_toolbar:
custom_inline_toolbar:
ep_custom_inline_toolbar/static/js/index
ep_font_color:
main:
ep_font_color/static/js/index
ep_font_size:
main:
ep_font_size/static/js/index
ep_image_upload:
ep_image_upload:
ep_image_upload/static/js/toolbar
ep_readonly_guest:
ep_readonly_guest:
ep_readonly_guest/static/js/client
userJoinOrUpdate:
ep_announce:
ep_announce:
ep_announce/static/js/announce:announce.userJoinOrUpdate
ep_webrtc:
ep_webrtc:
ep_webrtc/static/js
userLeave:
ep_announce:
ep_announce:
ep_announce/static/js/announce:announce.userLeave
ep_webrtc:
ep_webrtc:
ep_webrtc/static/js
```
